test/sinon/spy_test.js

```diff
@@ -901,6 +901,72 @@ if (typeof require === "function" && typeof module === "object") {
             }
         },
 
+        ".argNames": {
+            //jscs:disable disallowDanglingUnderscores
```

Comment:
I disable the "dangling underscore" lint rule for the just these tests. Otherwise this lint rule actually helps ensure that aliased arguments will never collide with existing api properties.